clar2wasm/tests/wasm-generation/functions.rs

```diff
@@ -0,0 +1,60 @@
+use clar2wasm::tools::crosscheck;
+use clarity::vm::Value;
+use proptest::proptest;
+use proptest::strategy::Strategy;
+
+use crate::{random_expressions, PropValue, TypePrinter};
+
+const DEFINE_PRIVATE_READONLY: [&str; 2] = ["define-private", "define-read-only"];
+
+proptest! {
+    #![proptest_config(super::runtime_config())]
+
+    #[test]
+    fn define_private_readonly(
```

Comment:
Why would you test two separate Clarity functions in the same test? If the test breaks, how are we suppose to know which of the two functions is wrong?

Reply:
I think [this](https://github.com/stacks-network/clarity-wasm/blob/ed807ff4bfe7d7279827c5b1ec02f4a9e79fe141/clar2wasm/tests/wasm-generation/arithmetic.rs#L40) is similar

Reply:
yes, it is as bad. No idea what would happen if the test failed.